README.md

```diff
@@ -34,76 +34,57 @@ sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
 kubectl apply -f ./kubemacpool.yaml
 ```
 
-### Opting-in to kubemacpool service
+### kubemacpool service
 
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt in vms mac allocation in your namespace
+Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms using two webhooks: mutatepods and mutatevirtualmachines.
```

Comment:
The fact that they are webhooks is irrelevant to a user. You can drop that part.

pods/vms -> Pods and VirtualMachines

Reply:
Done